test/e2e/specs/editor/blocks/code.spec.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+
+test.describe( 'Code', () => {
+	test.beforeEach( async ({pageUtils}) => {
+		await pageUtils.createNewPost();
+	} );
+
+	test( 'can be created by three backticks and enter', async ({page,pageUtils}) => {
+		await pageUtils.clickBlockAppender();
```

Comment:
We usually want to limit the usage of e2e utils if we can inline them directly. If we're just creating a page, I believe we can just click on the appender button directly:

```suggestion
		await page.click( 'role=button[name="Add default block"i]' );
```

The same code has also been used in `copy-cut-paste` tests. This way we don't have to create the `clickBlockAppender` util (unless we really need it in other situations).